src/overlays/actors/ovl_En_Sc_Ruppe/z_en_sc_ruppe.c

```diff
@@ -15,10 +16,22 @@ void EnScRuppe_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnScRuppe_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnScRuppe_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80BD6A8C(EnScRuppe* this, GlobalContext* globalCtx);
 void func_80BD6B18(EnScRuppe* this, GlobalContext* globalCtx);
 
-#if 0
+typedef enum {
+    /* 0 */ RUPPE_GREEN,
+    /* 1 */ RUPPE_BLUE,
+    /* 2 */ RUPPE_RED,
+    /* 3 */ RUPPE_ORANGE,
+    /* 4 */ RUPPE_PURPLE,
+    /* 5 */ RUPPE_SILVER,
```

Comment:
We realised in Discord that this can't be silver because those are 100 in MM.. maybe just say "unused".

This should also go in the header since it can be used by other actors that spawn this one, or in scene files once we commit parts of them.